mtgjson5/providers/tcgplayer.py

```diff
@@ -471,16 +494,51 @@ def get_tcgplayer_prices_map(
     return prices_map
 
 
-def convert_sku_data_enum(sku: Dict[str, int]) -> Dict[str, Union[int, str]]:
+def get_card_finish(card_name: str) -> Optional[str]:
+    """
+    Determine a card's TCGPlayer finish based on the card name,
+    as TCGPlayer indicates their finishes by ending a card's name
+    with "(Finish)". This can be a bit wonky for some edge cases,
+    but overall this should be good enough.
+    :param card_name: Card name from TCGPlayer
+    :return Card finish, if one is found
+    """
+    result_card_finish = None
+
+    card_finishes = re.findall(r"\(([^)0-9]+)\)", card_name)
+    for card_finish in card_finishes:
+        if not CardFinish.has_value(card_finish):
+            LOGGER.warning(f"Unknown TCGPlayer card finish: {card_finish}")
+            continue
+
+        result_card_finish = CardFinish(card_finish).name.replace("_", " ")
+        break
+
+    return result_card_finish
+
+
+def convert_sku_data_enum(product: Dict[str, Any]) -> List[Dict[str, Union[int, str]]]:
     """
-    Converts a TCGPlayer SKU from IDs to components
-    :param sku: TCGPlayer SKU component
-    :return: Enhanced TCGPlayer SKU dict
+    Converts a TCGPlayer Product's SKUs from IDs to components
+    :param product: TCGPlayer Product
+    :return: Enhanced List of TCGPlayer SKU dict objects
     """
-    return {
-        "skuId": sku["skuId"],
-        "productId": sku["productId"],
-        "language": CardLanguage(sku["languageId"]).name.replace("_", " "),
-        "printing": CardPrinting(sku["printingId"]).name.replace("_", " "),
-        "condition": CardCondition(sku["conditionId"]).name.replace("_", " "),
-    }
+    results = []
+
+    name = product["name"]
+    card_finish = get_card_finish(name)
+
+    skus = product["skus"]
+    for sku in skus:
+        entry = {
+            "skuId": sku["skuId"],
+            "productId": sku["productId"],
+            "language": CardLanguage(sku["languageId"]).name.replace("_", " "),
+            "printing": CardPrinting(sku["printingId"]).name.replace("_", " "),
+            "condition": CardCondition(sku["conditionId"]).name.replace("_", " "),
+        }
+        if card_finish:
+            entry["finish"] = card_finish
```

Comment:
maybe make it all caps like the other fields?

Reply:
It is